alpine/main/bootstrap.sh

```diff
@@ -1,21 +1,18 @@
 #!/bin/bash
-set -e
+# bootstrap clam av service and clam av database updater shell script
+# presented by mko (Markus Kosmal<dude@m-ko.de>)
+set -m
 
-if [[ ! -z "${FRESHCLAM_CONF_FILE}" ]]; then
-    echo "[bootstrap] FRESHCLAM_CONF_FILE set, copy to /etc/clamav/freshclam.conf"
-    mv /etc/clamav/freshclam.conf /etc/clamav/freshclam.conf.bak
-    cp -f ${FRESHCLAM_CONF_FILE} /etc/clamav/freshclam.conf
-fi
-
-if [[ ! -z "${CLAMD_CONF_FILE}" ]]; then
-    echo "[bootstrap] CLAMD_CONF_FILE set, copy to /etc/clamav/clamd.conf"
-    mv /etc/clamav/clamd.conf /etc/clamav/clamd.conf.bak
-    cp -f ${CLAMD_CONF_FILE} /etc/clamav/clamd.conf
-fi
+# configure freshclam.conf and clamd.conf from env variables if present
+source /envconfig.sh
 
 if ! [ -z $HTTPProxyServer ]; then echo "HTTPProxyServer $HTTPProxyServer" >> /etc/clamav/freshclam.conf; fi && \
 if ! [ -z $HTTPProxyPort   ]; then echo "HTTPProxyPort $HTTPProxyPort" >> /etc/clamav/freshclam.conf; fi && \
 
+DB_DIR=$(sed -n 's/^DatabaseDirectory\s\(.*\)\s*$/\1/p' /etc/clamav/freshclam.conf )
```

Comment:
Why did you double the DB_DIR part? It's redundant now, isn't it?

Reply:
Good catch. It wasn't on purpose.
I've just removed the doubled parts.